### PR TITLE
chore: update README with overview and reference to examples

### PR DESCRIPTION
**Pull request by sasha-gitg**

### CODE REVIEW

**Review by dizcology — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -69,6 +69,254 @@ Windows
     <your-env>\Scripts\activate
     <your-env>\Scripts\pip.exe install google-cloud-aiplatform
 
+
+Overview
+~~~~~~~~
+This section provides a brief overview of the Vertex SDK for Python. You can also reference the notebooks in `vertex-samples`_ for examples.
```
Comment (telpirion) [inline README.rst:75]: Insert 'AI' after Vertex.

File: README.rst [hunk 2]
```diff
@@ -69,6 +69,254 @@ Windows
...
+SDK functionality can be used from the root of the package:
+
+.. code-block:: Python
+
+    from google.cloud import aiplatform
+
+
+Initialization
+^^^^^^^^^^^^^^
+Initialize the SDK to store common configurations that will be used throughout the SDK.
```
Comment (telpirion) [inline README.rst:90]: NIT: Rewrite in present tense, second person.

Suggestion:

```
Initialize the SDK to store common configurations that you use with the SDK. 
```

File: README.rst [hunk 3]
```diff
@@ -69,6 +69,254 @@ Windows
...
+
+.. code-block:: Python
+
+    aiplatform.init(
+        # your GCP project ID or number
```
Comment (telpirion) [inline README.rst:95]: Replace with "Google Cloud Project." Avoid abbreviations like GCP, GCS, etc. whenever possible.

File: README.rst [hunk 4]
```diff
@@ -69,6 +69,254 @@ Windows
...
+        # logged metrics and parameters
+        experiment='my-experiment',
+
+        # description of the experiment above
+        experiment_description='my experiment decsription' 
+    )
+
+Datasets
+^^^^^^^^
+AI Platform provides managed Tabular, Text, Image, and Video datasets. In the SDK, Datasets can be used downstream to
```
Comment (telpirion) [inline README.rst:125]: Vertex AI

Comment (telpirion) [inline README.rst:125]: Remove initial caps. These aren't proper nouns.

Comment (telpirion) [inline README.rst:125]: Remove initial caps, unless you're referring to a `Dataset` class (then it's a proper noun).

File: README.rst [hunk 5]
```diff
@@ -69,6 +69,254 @@ Windows
...
+train models.
+
+To create a Tabular dataset:
```
Comment (telpirion) [inline README.rst:128]: Remove initial caps. We don't treat these as proper nouns in the documentation (see [here](https://cloud.google.com/vertex-ai/docs/datasets/prepare-tabular))

File: README.rst [hunk 6]
```diff
@@ -69,6 +69,254 @@ Windows
...
+
+Vertex AI supports a variety of dataset schemas. References to these schemas are available under the
+:code:`aiplatform.schema.dataset` namespace. For more information on the supported dataset schemas please refer to the
+`Preparing data docs`_.
+
+.. _Preparing data docs: https://cloud.google.com/ai-platform-unified/docs/datasets/prepare
+
+Training
+^^^^^^^^
+The Vertex SDK allows you train Custom and AutoML Models.
```
Comment (telpirion) [inline README.rst:163]: Insert 'AI' after "Vertex."

File: README.rst [hunk 7]
```diff
@@ -69,6 +69,254 @@ Windows
...
+
+Custom models can be trained using a custom Python script, custom Python package, or container.
```
Comment (telpirion) [inline README.rst:165]: NIT: Rephrase in active voice.

Suggestion:

```

You can train custom models using a custom Python script, custom Python package, or container.
```

File: README.rst [hunk 8]
```diff
@@ -69,6 +69,254 @@ Windows
...
+
+Preparing Your Custom Code
+--------------------------
+Vertex AI custom training enables you to train on AI Platform Datasets and produce AI Platform Models. To do so your
```
Comment (telpirion) [inline README.rst:169]: Does this refer to legacy AI Platform?

Comment (telpirion) [inline README.rst:169]: "Models" and "Datasets" aren't proper nouns--remove initial capitalization.

File: README.rst [hunk 9]
```diff
@@ -69,6 +69,254 @@ Windows
...
+      model_serving_container_image_uri="gcr.io/cloud-aiplatform/prediction/tf2-cpu.2-2:latest",
+  )
+
+  model = job.run(my_dataset,
+                  replica_count=1,
+                  machine_type="n1-standard-4",
+                  accelerator_type='NVIDIA_TESLA_K80',
+                  accelerator_count=1)
+
+In the code block above my_dataset is managed dataset created in the `Datasets` section above. `model` is a Managed Vertex AI Model that the be deployed or exported.
```
Comment (telpirion) [inline README.rst:210]: Put in code font.

Comment (telpirion) [inline README.rst:210]: NIT: Use an article with this variable.

Suggestion:

```
The `model` variable is a managed Vertex AI model that can be deployed or exported.
```

File: README.rst [hunk 10]
```diff
@@ -69,6 +69,254 @@ Windows
...
+
+
+AutoMLs
```
Comment (telpirion) [inline README.rst:213]: Remove 's'. Product names shouldn't be pluralized (unless it's part of the product name).

File: README.rst [hunk 11]
```diff
@@ -69,6 +69,254 @@ Windows
...
+-------
+The Vertex SDK for Python supports AutoML Tabular, Image, Text, Video, and Forecasting.
```
Comment (telpirion) [inline README.rst:215]: Remove initial caps from "Tabular," "Image," "Text," "Video," and "Forecasting"--these aren't proper nouns.

File: README.rst [hunk 12]
```diff
@@ -69,6 +69,254 @@ Windows
...
+
+To train an AutoML Tabular model:
```
Comment (telpirion) [inline README.rst:217]: Remove initial caps on "Tabular."

File: README.rst [hunk 13]
```diff
@@ -69,6 +69,254 @@ Windows
...
+
+  endpoint = model.deploy(machine_type="n1-standard-4",
+                          min_replica_count=1,
+                          max_replica_count=5
+                          machine_type='n1-standard-4',
+                          accelerator_type='NVIDIA_TESLA_K80',
+                          accelerator_count=1)
+
+
+To upload a Model:
```
Comment (telpirion) [inline README.rst:257]: Remove initial caps.

File: README.rst [hunk 14]
```diff
@@ -69,6 +69,254 @@ Windows
...
+
+.. code-block:: Python
+
+  model = aiplatform.Model.upload(
+      display_name='my-model',
+      artifact_uri="gs://python/to/my/model/dir",
+      serving_container_image_uri="gcr.io/cloud-aiplatform/prediction/tf2-cpu.2-2:latest",
+  )
+
+To get a Model:
```
Comment (telpirion) [inline README.rst:267]: Remove initial caps.

File: README.rst [hunk 15]
```diff
@@ -69,6 +69,254 @@ Windows
...
+---------
+
+To get predictions from endpoints:
+
+.. code-block:: Python
+
+  endpoint.predict(instances=[[6.7, 3.1, 4.7, 1.5], [4.6, 3.1, 1.5, 0.2]])
+
+
+To create an Endpoint
```
Comment (telpirion) [inline README.rst:288]: Remove initial caps.

File: README.rst [hunk 16]
```diff
@@ -69,6 +69,254 @@ Windows
...
+
+.. code-block:: Python
+
+  endpoint = endpoint.create(display_name='my-endpoint')
+
+To deploy a Model to a created Endpoint:
```
Comment (telpirion) [inline README.rst:294]: Remove initial caps.

**Review by telpirion — APPROVED** [review]
Some style concerns and a few nits. Thank you!

File: README.rst (hunk 6 above)
Comment (sasha-gitg) [reply README.rst:163]: Changed to Vertex SDK for Python. See here: https://cloud.google.com/vertex-ai/docs/start/client-libraries#python

File: README.rst (hunk 8 above)
Comment (sasha-gitg) [reply README.rst:169]: No, changed to Vertex AI

**Pull request merged**